### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by Jean20B**
I don't know exactly why there are old updates but that should not be a problem ^^'

### CODE REVIEW

File: fr_FR/LC_MESSAGES/search.po [hunk 1]
```diff
@@ -29,17 +29,17 @@ msgid "RESULTS_FOUND_FORMAT_MULTIPLE"
...
-msgstr "Flipnote ID"
+msgstr "@ID Flipnote"
 
 msgid "SHORT_KEY_NO_RESULTS"
-msgstr "No Flipnote found"
+msgstr "Aucune Flipnote trouvée"
 
 msgid "SHORT_KEY_FORMAT_GUIDE"
-msgstr "Not finding what you're looking for? You may have entered the Flipnote ID incorrectly. Flipnote IDs are six digits long and can be found in the details section for the Flipnote."
+msgstr "Vous ne trouvez pas ce que vous cherchez ? Vous avez peut-être mal écrit l'ID de la Flipnote. Les ID Flipnotes contiennent 6 chiffres et peuvent être trouvées dans les détails de la Flipnote."
```
Comment (sudofox) [inline fr_FR/LC_MESSAGES/search.po]: There's a space after 'cherchez' -- can you remove that and push the change in one more commit?

**Comment by sudofox** [comment]
@Jean20B See the review of search.po -- it's a really simple change -- and let me know when that's updated.

**Comment by sudofox** [comment]
actually I'll just make the change myself

File: fr_FR/LC_MESSAGES/search.po (hunk 1 above)
Comment (Jean20B) [reply fr_FR/LC_MESSAGES/search.po]: In French there's always a space before "!" and "?"

**Pull request merged**